### PR TITLE
Kinto init command

### PR DESCRIPTION
**Pull request by Natim**
Superseeded #271

### CODE REVIEW

File: kinto/config/__init__.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    values['secret'] = binascii.b2a_hex(os.urandom(32))
+
+    backend = input("Which backend to use? "
+                    "(1 - postgresql, 2 - redis, default - memory) ").strip()
+
+    if backend == '1':
+        # Postgresql configuration
+        values['storage_backend'] = "cliquet.storage.postgresql"
+        values['storage_url'] = ("postgres://postgres:postgres@localhost/"
+                                 "postgres")
```
Comment (leplatrem) [inline kinto/config/__init__.py]: I would put this URI in a variable instead of repeating it 3 times

File: kinto/config/__init__.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        with codecs.open(destination, 'w+', encoding='utf-8') as output:
+            output.write(rendered)
+
+
+def init(config_file):
+    values = {}
+    values['secret'] = binascii.b2a_hex(os.urandom(32))
+
+    backend = input("Which backend to use? "
+                    "(1 - postgresql, 2 - redis, default - memory) ").strip()
```
Comment (leplatrem) [inline kinto/config/__init__.py]: What happen if user types 3 :) ?

File: kinto/config/__init__.py (hunk 1 above)
Comment (Natim) [inline kinto/config/__init__.py]: It is configuring the memory backend.

Comment (Natim) [inline kinto/config/__init__.py]: Ok

Comment (leplatrem) [inline kinto/config/__init__.py]: Yes I can read that. But we don't complain?

File: kinto/config/__init__.py (hunk 2 above)
Comment (leplatrem) [inline kinto/config/__init__.py]: We don't complain?

Comment (Natim) [inline kinto/config/__init__.py]: The idea was to default to memory but we can discuss this if you think otherwise.

**Comment by leplatrem** [comment]
Ok great ! Let's go through some missing parts:

* [ ] Add a rule in Makefile to run init when config file is missing (like in #261)
* [ ] Add default config/kinto.ini to .gitignore
* [ ] Update documentation to remove wget of remote ini files
* [ ] Update quickstart to mention this command
* [ ] Remove `cp config/kinto.ini config/dev.ini` from documentation
* [ ] Warn (or create backup or whatever) when destination file already exists

File: kinto/config/__init__.py (hunk 2 above)
Comment (leplatrem) [inline kinto/config/__init__.py]: default when empty string is seized yes, but surrounding the input with a `while True: ... if backend in ('', '1', '2'): break` would not change much

**Comment by leplatrem** [comment]
r+

**Pull request merged**